deployments/base-goerli/usdc/migrations/1679592519_configurate_and_ens.ts

```diff
@@ -0,0 +1,209 @@
+import { DeploymentManager } from '../../../../plugins/deployment_manager/DeploymentManager';
+import {diffState, getCometConfig} from '../../../../plugins/deployment_manager/DiffState';
+import { migration } from '../../../../plugins/deployment_manager/Migration';
+import { calldata, exp, getConfigurationStruct, proposal } from '../../../../src/deploy';
+import { expect } from 'chai';
+
+const ENSName = 'compound-community-licenses.eth';
+const ENSResolverAddress = '0x19c2d5D0f035563344dBB7bE5fD09c8dad62b001';
+const ENSRegistryAddress = '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e';
+const ENSSubdomainLabel = 'v3-additional-grants';
+const ENSSubdomain = `${ENSSubdomainLabel}.${ENSName}`;
+const ENSTextRecordKey = 'v3-official-markets';
+const baseCOMPAddress = '0xA29b548056c3fD0f68BAd9d4829EC4E66f22f796';
+
+export default migration('1679592519_configurate_and_ens', {
+  prepare: async (deploymentManager: DeploymentManager) => {
+    return {};
+  },
+
+  enact: async (deploymentManager: DeploymentManager, govDeploymentManager: DeploymentManager) => {
+    const trace = deploymentManager.tracer();
+    const ethers = deploymentManager.hre.ethers;
+    const { utils } = ethers;
+
+    const {
+      bridgeReceiver,
+      comet,
+      cometAdmin,
+      configurator,
+      rewards,
+    } = await deploymentManager.getContracts();
+
+    const {
+      baseL1CrossDomainMessenger,
+      baseL1StandardBridge,
+      governor,
+      COMP: goerliCOMP,
+    } = await govDeploymentManager.getContracts();
+
+    // ENS Setup
+    // See also: https://docs.ens.domains/contract-api-reference/name-processing
+    const ENSResolver = await govDeploymentManager.existing('ENSResolver', ENSResolverAddress, 'goerli');
+    const subdomainHash = ethers.utils.namehash(ENSSubdomain);
+    const baseGoerliChainId = (await deploymentManager.hre.ethers.provider.getNetwork()).chainId.toString();
```

Comment:
any reason not to just hardcode in the Base Goerli chain id?

Reply:
Not really, just one less place to hardcode things. We do check that it is set correctly in the `verify` step.